Jenkinsfile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env groovy
-common {
-  slackChannel = '#clients-eng'
-  upstreamProjects = 'confluentinc/rest-utils'
+
+docker_oraclejdk8 {
```

Comment:
why the change to jdk8 here? we'l lose jdk7 correctness checks if we do this. 4.0.x seems like the wrong place to make that change. (presumably we'd move it only after 4.1.x branches are created)

Reply:
It's an unfortunately named Pipeline library.

Down a bit in the Jenkinsfile, `nodeLabel = 'docker-oraclejdk7'` puts us back on a Java 7 build node. We just need the `docker_oraclejdk8` library to set up the Docker env for us.

Reply:
yeah, seems to be a bit of conflation of naming/platform being tested on with the jenkinsfile setup. not sure if there's really a fix to this since aside from overriding the value here actually does define a lot about how the job runs. but this is definitely at least confusing here.